corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py

```diff
@@ -12,16 +12,15 @@
 logger = logging.getLogger('accounting')
 
 
-def bootstrap_twilio_gateway(apps):
+def bootstrap_twilio_gateway(apps, twilio_rates_filename):
     currency_class = apps.get_model('accounting', 'Currency') if apps else Currency
     sms_gateway_fee_class = apps.get_model('smsbillables', 'SmsGatewayFee') if apps else SmsGatewayFee
     sms_gateway_fee_criteria_class = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria') if apps else SmsGatewayFeeCriteria
 
     # iso -> provider -> rate
     def get_twilio_data():
-        twilio_file = open('corehq/apps/smsbillables/management/'
-                           'commands/pricing_data/twilio-international-sms-rates.csv')
-        twilio_csv = csv.reader(twilio_file)
+        twilio_file = open(twilio_rates_filename)
+        twilio_csv = csv.reader(twilio_file.read().splitlines())
```

Comment:
not following why splitlines is necessary now - doesn't `reader` take in a file and return an iterable?